ecs-init/docker/dependencies_test.go

```diff
@@ -1,6 +1,3 @@
-//go:build test
```

Comment:
these tests were not running because of these flags, so some had to be fixed as they havent been passing for a long time (or possibly not ever)

Reply:
almost none of the test files in the `ecs-init/` directory use the `-tags test` flag, I'm guessing this is only here from someone copy and pasting the file header from the `agent/` directory.